front-end/src/components/newsfeed-view/Backgrounds.scss

```diff
@@ -0,0 +1,12 @@
+.backgrounds {
+    height: 55em;
+    width: 100%;
+    position: absolute;
+    z-index: -1;
+    top: 0;
+    left: 0;
+    background-image: url("../../assets/images/bg1.jpg");
```

Comment:
this wouldnt really work. webpack moves path around and shorten files. If you bypass it (from css, cuz webpack is js), this could fail. I dont know for sure but people advised me to pass this as props, and inject it from the javascript file.